tmt/base.py

```diff
@@ -658,33 +658,11 @@ def _export(self, *, keys: Optional[List[str]] = None) -> tmt.export._RawExporte
             elif isinstance(value, FmfId):
                 data[key] = value.to_dict()
 
-            # TODO: this belongs to Test.export, and it will be moved when the time
-            # of export() cleanup comes.
-            elif key in ('require', 'recommend') and value:
-                data[key] = [
-                    require.to_minimal_spec() for require in cast(List[Require], value)
-                    ]
-
-            # TODO: this belongs to Story.export, and it will be moved when the time
-            # of export() cleanup comes.
-            elif key == 'priority' and value is not None:
-                data[key] = cast(StoryPriority, value).value
-
             # TODO: this belongs to Plan.export, and it will be moved when the time
             # of export() cleanup comes.
             elif key == 'script' and isinstance(value, list):
                 data[key] = [str(script) for script in cast(List[ShellScript], value)]
```

Comment:
@psss would you be able to recall what this field might be? I can't find any `script` field in **test** metadata, the best fit seems to be the `prepare` or `finish` `shell` plugin, but that's not what's being exported here...

Reply:
Isn't `script` field general for more steps? I hope I'm not completely wrong. I need to dive in more to the code base.

Reply:
Hm, this is weird. Except for `prepare` and `finish` I'd say the only place where `script` is used is the super simple `execute` step scenario:

```yaml
summary: Basic smoke test
execute:
    script: foo --version
```

But even there, `script` is under `execute`, not as the top plan field.

Reply:
It does smell like a mistake, maybe I created it when I was moving fields around. I'd go ahead and remove it.

Reply:
Agreed, ack.